README.md

```diff
@@ -103,7 +97,6 @@ can talk to each other...
 
 On `$HOST1`...
```

Comment:
"In the container started on $HOST1" - yes, if you actually run the example it is obvious that the following command runs inside the container, but that's not so clear when just casually reading the docs.